src/layouts/Contact-HEDA Contact Layout.layout

```diff
@@ -86,6 +86,10 @@
                 <behavior>Edit</behavior>
                 <field>Financial_Aid_Applicant__c</field>
             </layoutItems>
+            <layoutItems>
```

Comment:
I guessed where this field should be. I should review this with the PMs in a meeting next week.